showcase/src/app/shared/table-of-contents/table-of-contents.component.spec.ts

```diff
@@ -0,0 +1,45 @@
+import { async, ComponentFixture, TestBed } from "@angular/core/testing";
+import { TableOfContentsComponent } from "./table-of-contents.component";
+import { NO_ERRORS_SCHEMA } from "@angular/core";
+
+describe("TableOfContents", () => {
+	let fixture: ComponentFixture<TableOfContentsComponent>;
+	let component: TableOfContentsComponent;
+
+	beforeEach(async(() => {
+		return TestBed.configureTestingModule({
+			declarations: [TableOfContentsComponent],
+			imports: [],
+			schemas: [NO_ERRORS_SCHEMA] // tells the Angular compiler to ignore unrecognized elements and attributes: mat-icon
+		}).compileComponents();
+	}));
+
+	beforeEach(() => {
+		fixture = TestBed.createComponent(TableOfContentsComponent);
+		component = fixture.componentInstance;
+		fixture.detectChanges();
+	});
+
+	fit("should have no header", () => {
```

Comment:
It seems you missed to remove the focus on this test 😋

Reply:
Oooh sh*t sorry ! I'll remove that first thing tomorrow ;)